index.html

```diff
@@ -646,8 +646,8 @@ <h3>Versions of guidance</h3>
 		</section>
 		<section> <!-- User Experience Design -->
 			<h2>User Experience Design</h2>
-			<p><strong>If you're creating content and systems designed for users, then whether you know it or not, you're working in user experience.</strong></p>
-			<p>Good user experience reduces time and (data) resources wasted on the journey. Bad user experience design does the opposite, while potentially also having a negative impact on accessibility. Meanwhile, the aesthetic and visual choices - and how we choose to present information - can have some of the biggest impact on asset size, performance, and overall web sustainability. </p>
+			<p><strong>If you are creating content and systems designed for users, then whether you know it or not, you are working in user experience (<abbr title="User Experience">UX</abbr>).</strong></p>
```

Comment:
I the `abbr` necessary here considering the abbreviation is right next to the words?